trade-finance-manager-api/src/v1/controllers/amendment.controller-updateFacilityAmendment.test.js

```diff
@@ -0,0 +1,141 @@
+        firstName: underwriter.firstName,
+        lastName: underwriter.lastName,
+      },
+    };
+
+    it('should call getTasksAssignedToUserByGroup if request contains leadUnderwriter', async () => {
+      // Arrange
+      const { req, res } = createMocks({ params: { amendmentId, facilityId }, user: underwriter, body: requestBody });
+
+      // Act
+      await amendmentController.updateFacilityAmendment(req, res);
+
+      // Assert
+      expect(createTasksAmendmentHelper.getTasksAssignedToUserByGroup).toHaveBeenCalledTimes(1);
+      expect(createTasksAmendmentHelper.getTasksAssignedToUserByGroup).toHaveBeenCalledWith(
+        MOCK_AMENDMENT.tasks,
+        TEAMS.UNDERWRITERS.id,
+        requestBody.leadUnderwriter._id,
+      );
+    });
+
+    it("should not call getTasksAssignedToUserByGroup if request doesn't contains leadUnderwriter", async () => {
+      // Arrange
+      const { req, res } = createMocks({ params: { amendmentId, facilityId }, user: underwriter, body: {} });
+
+      // Act
+      await amendmentController.updateFacilityAmendment(req, res);
+
+      // Assert
+      expect(createTasksAmendmentHelper.getTasksAssignedToUserByGroup).not.toHaveBeenCalled();
+    });
+
+    it('should call api.updateFacilityAmendment with tasks returned by getTasksAssignedToUserByGroup for Underwriter', async () => {
+      // Arrange
+      const { req, res } = createMocks({ params: { amendmentId, facilityId }, user: underwriter, body: requestBody });
+
+      getTasksAssignedToUserByGroupMock = jest.spyOn(createTasksAmendmentHelper, 'getTasksAssignedToUserByGroup');
+      getTasksAssignedToUserByGroupMock.mockResolvedValue(TASKS_ASSIGNED_TO_UNDERWRITER);
+
+      const auditDetails = { id: underwriter._id, userType: 'tfm' };
+
+      // Act
+      await amendmentController.updateFacilityAmendment(req, res);
+
+      // Assert
+      expect(api.updateFacilityAmendment).toHaveBeenCalledTimes(1);
+      expect(api.updateFacilityAmendment).toHaveBeenCalledWith(facilityId, amendmentId, { ...requestBody, tasks: TASKS_ASSIGNED_TO_UNDERWRITER }, auditDetails);
+    });
+
+    it('should call api.updateFacilityAmendment with tasks returned by getTasksAssignedToUserByGroup for Underwriter manager', async () => {
+      // Arrange
+      const requestBodyWithUnderwriterManager = {
+        leadUnderwriter: {
+          _id: underwriterManager._id,
+          firstName: underwriterManager.firstName,
+          lastName: underwriterManager.lastName,
+        },
+      };
+      const { req, res } = createMocks({
+        params: { amendmentId, facilityId },
+        user: underwriterManager,
+        body: requestBodyWithUnderwriterManager,
+      });
+
+      getTasksAssignedToUserByGroupMock = jest.spyOn(createTasksAmendmentHelper, 'getTasksAssignedToUserByGroup');
+      getTasksAssignedToUserByGroupMock.mockResolvedValue(TASKS_ASSIGNED_TO_UNDERWRITER_MANAGER);
+
+      const auditDetails = { id: underwriterManager._id, userType: 'tfm' };
```

Comment:
Same as above.

Reply:
'tfm' string is used a lot in auditDetails functionality, introducing constant would be quite big touch to auditDetails code. This PR is not related to auditDetails, so lets leave 'tfs' as a string? 

https://github.com/UK-Export-Finance/dtfs2/blob/main/libs/common/src/types/audit-details.ts